packages/cli/package.json

```diff
@@ -40,9 +40,9 @@
     "typescript": "^3.7.0"
   },
   "dependencies": {
-    "@bigtest/effection": "^0.4.0",
-    "@bigtest/project": "^0.4.0",
-    "@bigtest/server": "^0.4.0",
+    "@bigtest/effection": "^0.5.0",
+    "@bigtest/project": "^1.0.0",
+    "@bigtest/server": "^1.0.0",
```

Comment:
Looks like we have a major version bump in here somewhere? Server API is likely to change pretty heavily over the next several months. Is it wise to go 1.0 now?